py_proxy/templates/index.html.jinja2

```diff
@@ -1,6 +1,99 @@
 <html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>via.hypothes.is</title>
+  <style>
+    @import url("//unpkg.com/normalize.css");
+
+    body {
+      font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+      height: 100vh;
+    }
+
+    .content {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+      justify-content: center;
+      max-height: 70vh;
+    }
+
+    .url-form {
+      align-items: center;
+      display: flex;
+      flex-direction: column;
+      max-width: 400px;
+      width: 95%;
+    }
+
+    /* Input field for URL to annotate. */
+    .url-field {
+      align-self: stretch;
+      border-radius: 3px;
+      border: 1px solid #ccc;
+      font-size: 20px;
+      margin-bottom: 20px;
+      margin-top: 20px;
+      min-height: 45px;
+      padding-bottom: 5px;
+      padding-left: 16px;
+      padding-right: 16px;
+      padding-top: 5px;
+    }
+
+    .url-field:focus {
+      border-color: #3f3f3f;
+    }
+
+    /* Button that takes the user to the proxied page. */
+    .annotate-btn {
+      background-color: #3f3f3f;
+      border-radius: 3px;
+      border: none;
+      color: #fff;
+      cursor: pointer;
+      font-size: 20px;
+      font-weight: 700;
+      max-width: 100px;
+      padding-bottom: 10px;
+      padding-left: 15px;
+      padding-right: 15px;
+      padding-top: 10px;
+    }
+
+    .annotate-btn:hover {
+      background-color: #d00032;
+    }
+  </style>
+</head>
 <body>
-    <a> Hello World </a>
+  <div class="content">
+  <a href="https://hypothes.is">
+    <svg xmlns="http://www.w3.org/2000/svg"
+         width="100"
+         height="100"
+         viewBox="0 0 24 28"
+         aria-labelledby="logo-title">
+      <title id="logo-title">Hypothesis homepage</title>
+      <g fill="none" fill-rule="evenodd"><path fill="#FFF" fill-rule="nonzero" d="M3.886 3.945H21.03v16.047H3.886z"/><path d="M0 2.005C0 .898.897 0 2.005 0h19.99C23.102 0 24 .897 24 2.005v19.99A2.005 2.005 0 0 1 21.995 24H2.005A2.005 2.005 0 0 1 0 21.995V2.005zM9 24h6l-3 4-3-4zM7.008 4H4v16h3.008v-4.997C7.008 12.005 8.168 12.01 9 12c1 .007 2.019.06 2.019 2.003V20h3.008v-6.891C14.027 10 12 9.003 10 9.003c-1.99 0-2 0-2.992 1.999V4zM19 19.987c1.105 0 2-.893 2-1.994A1.997 1.997 0 0 0 19 16c-1.105 0-2 .892-2 1.993s.895 1.994 2 1.994z" fill="#3F3F3F"/></g>
+    </svg>
+  </a>
+  <form class="url-form"
+        onsubmit="url = document.getElementById('search').value; if (url != '') { document.location.href = '/' + url; } return false;"
+        novalidate>
+    <input id="search"
+           aria-label="Web or PDF document to annotate"
+           autofocus
+           class="url-field"
+           name="search"
+           placeholder="Paste a link to create or view annotations"
+           type="url"
+           >
+    <button type="submit" class="annotate-btn">Go!</button>
```

Comment:
```suggestion
    <button type="submit" class="annotate-btn">Annotate!</button>
```

Another sneaky change to Via's UI. "Go" is not a good label for a button ("Go" could mean literally anything)

Reply:
This would also require a small CSS fix though. Not critical